documentation/src/docs/asciidoc/user-guide/running-tests.adoc

```diff
@@ -254,24 +254,23 @@ additional dependency to the runtime classpath.
 [[running-tests-build-maven]]
 ==== Maven
 
-Starting with https://issues.apache.org/jira/browse/SUREFIRE-1330[version 2.22.0], Maven
-Surefire and Maven Failsafe provide
+Maven Surefire and Maven Failsafe provide
 https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit-platform.html[native support]
 for executing tests on the JUnit Platform. The `pom.xml` file in the
 `{junit5-jupiter-starter-maven}` project demonstrates how to use the Maven Surefire plugin
 and can serve as a starting point for configuring your Maven build.
 
 [WARNING]
-.Use Maven Surefire/Failsafe 3.0.0-M4 or later to avoid interoperability issues
+.Use recent version of Maven Surefire/Failsafe to avoid interoperability issues
 ====
-Maven Surefire/Failsafe 3.0.0-M4
-https://issues.apache.org/jira/browse/SUREFIRE-1585[introduced support] for aligning the
-version of the JUnit Platform Launcher it uses with the JUnit Platform version found on
-the test runtime classpath. Therefore, it is recommended to use version 3.0.0-M4 or later
-to avoid interoperability issues.
-
-Alternatively, you can add a test dependency on the matching version of the JUnit Platform
-Launcher to your Maven build as follows.
+To avoid interoperability issues, it is recommended to use a recent versions of Maven
+Surefire/Failsafe (3.0.0 or later) because it automatically aligns the used version of the
+JUnit Platform Launcher with the JUnit Platform version found on the test runtime
+classpath.
```

Comment:
```suggestion
To avoid interoperability issues, it is recommended to use a recent version of Maven
Surefire/Failsafe (3.0.0 or later) because it automatically aligns the version of the
JUnit Platform Launcher that is uses with the JUnit Platform version found on the test runtime
classpath.
```

Reply:
Done in a15e4466ab25767f7a619d11cc1b0005abd70edb